website/screens/components/container/code/ContainerCodePage.tsx

```diff
@@ -0,0 +1,482 @@
+            <DxcFlex direction="column" gap="0.25rem" alignItems="baseline">
+              <StatusTag status="Required">Required</StatusTag>children
+            </DxcFlex>
+          </td>
+          <td>
+            <TableCode>React.ReactNode</TableCode>
+          </td>
+          <td>Custom content inside the container.</td>
+          <td>-</td>
+        </tr>
+        <tr>
+          <td>display</td>
+          <td>
+            <TableCode>'block' | 'inline-block' | 'inline' | 'none'</TableCode>
+          </td>
+          <td>
+            Sets the <Code>display</Code> CSS property. See{" "}
+            <DxcLink
+              newWindow
+              href="https://developer.mozilla.org/en-US/docs/Web/CSS/display"
+            >
+              MDN
+            </DxcLink>{" "}
+            for further information. The set of values is limited to the ones
+            related to the outer display type. If you want to apply any pattern
+            from the inner box and how the children are laid out, we recommend
+            you to use the Flex and Grid components.
+          </td>
+          <td>
+            <TableCode>'block'</TableCode>
+          </td>
+        </tr>
+        <tr>
+          <td>float</td>
+          <td>
+            <TableCode>'left' | 'right' | 'none'</TableCode>
+          </td>
+          <td>
+            Sets the <Code>float</Code> CSS property. See{" "}
+            <DxcLink
+              newWindow
+              href="https://developer.mozilla.org/en-US/docs/Web/CSS/float"
+            >
+              MDN
+            </DxcLink>{" "}
+            for further information.
+          </td>
+          <td>
+            <TableCode>'none'</TableCode>
+          </td>
+        </tr>
+        <tr>
+          <td>height</td>
+          <td>
+            <TableCode>string</TableCode>
+          </td>
+          <td>
+            Sets the <Code>height</Code> CSS property. See{" "}
+            <DxcLink
+              newWindow
+              href="https://developer.mozilla.org/en-US/docs/Web/CSS/height"
+            >
+              MDN
+            </DxcLink>{" "}
+            for further information.
+          </td>
+          <td>
+            <TableCode>'auto'</TableCode>
+          </td>
+        </tr>
+        <tr>
+          <td>inset</td>
+          <td>
+            <ExtendedTableCode>{insetTypeString}</ExtendedTableCode>
+          </td>
+          <td>
+            Based on the CSS property <Code>inset</Code> this prop is a
+            shorthand that corresponds to the <Code>top</Code>,{" "}
+            <Code>right</Code>, <Code>bottom</Code>, and/or <Code>left</Code>{" "}
+            properties.
+          </td>
+          <td>
+            <TableCode>'auto'</TableCode>
+          </td>
+        </tr>
+        <tr>
+          <td>margin</td>
+          <td>
+            <TableCode>
+              'xxsmall' | 'xsmall' | 'small' | 'medium' | 'large' | 'xlarge' |
+              'xxlarge' | Space
```

Comment:
I can see your point but for consistency, we need to stick with our current approach. It would be a huge task to change every margin prop type description.